packages/react-core/src/components/MenuToggle/MenuToggle.tsx

```diff
@@ -6,6 +6,13 @@ import { BadgeProps } from '../Badge';
 
 export type MenuToggleElement = HTMLDivElement | HTMLButtonElement;
 
+export interface SplitButtonOptions {
+  /** Elements to display before the toggle button. When included, renders the menu toggle as a split button. */
+  items?: React.ReactNode[];
```

Comment:
I would actually make the items required for the object.  If you are going to use it, you should define items.